Avatar.css

```diff
@@ -0,0 +1,38 @@
+.container {
+  box-sizing: border-box;
+  text-align: center;
+  color: darkblue;
+}
+.flex-container {
+  display: flex;
+}
+
+.main {
+  
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.avatar {
+  border-radius: 80px;
+}
+
+.av-xl {
+  height: 200px;
+  width: 200px;
+  border-radius: 100px;
+}
+.av-md {
+  height: 6.25rem;
+  width: 6.25rem;
+}
+.av-sm {
+  height: 3.125rem;
+  width: 3.125rem;
+}
```

Comment:
try to be consistent. Use rems above 10px

Reply:
Updated with Rem's in all CSS Files